apis_ontology/models.py

```diff
@@ -766,6 +767,11 @@ class Person(PersonNameMixin, DescriptionMixin, StatusMixin, AbstractEntity):
     class Meta:
         verbose_name_plural = _("personen")
 
+    @classmethod
+    def get_or_create_uri(cls, uri):
+        print(f"using custom get_or_create_uri with {uri}")
```

Comment:
Would remove the `print` statements; any relevant logging while importing is done via loggers.